stix_shifter_modules/sentinelone/stix_translation/json/from_stix_map.json

```diff
@@ -0,0 +1,270 @@
+        "tgtFileSize"
+      ],
+      "hashes.MD5": [
+        "tgtFileMd5",
+        "tgtFileOldMd5"
+      ],
+      "hashes.'SHA-1'": [
+        "tgtFileSha1",
+        "tgtFileOldSha1"
+      ],
+      "hashes.'SHA-256'": [
+        "tgtFileSha256",
+        "tgtFileOldSha256"
+      ],
+      "parent_directory_ref.path": [
+        "tgtFilePath",
+        "tgtFileOldPath"
+      ],
+      "created": [
+        "tgtFileCreatedAt"
+      ],
+      "modified": [
+        "tgtFileModifiedAt"
+      ]
+    }
+  },
+  "directory": {
+    "fields": {
+      "path": [
+        "tgtFilePath",
+        "tgtFileOldPath"
+      ]
+    }
+  },
+  "user-account": {
+    "fields": {
+      "user_id": [
+        "loginAccountSid",
+        "srcProcUser",
+        "tgtProcUser"
+      ],
+      "account_login": [
+        "loginsUserName"
+      ],
+      "is_privileged": [
+        "loginIsAdministratorEquivalent"
+      ],
+      "display_name": [
+        "loginAccountName"
+      ]
+    }
+  },
+  "windows-registry-key": {
+    "fields": {
+      "key": [
+        "registryKeyPath"
+      ],
+      "values[*]": [
+        "registryValue"
+      ]
+    }
+  },
+  "x-sentinelone-process": {
+    "fields": {
+      "image_path": [
+        "srcProcImagePath",
+        "tgtProcImagePath"
+      ],
+      "integrity_level": [
+        "srcProcIntegrityLevel",
+        "tgtProcIntegrityLevel"
+      ],
+      "publisher": [
+        "srcProcPublisher",
+        "tgtProcPublisher"
+      ],
+      "parent_process_start_time": [
+        "srcProcParentStartTime"
+      ],
+      "parent_image_path": [
+        "srcProcParentImagePath"
+      ]
+    }
+  },
+  "x-sentinelone-network-action": {
+    "fields": {
+      "connection_status": [
+        "netConnStatus"
+      ]
+    }
+  },
+  "x-sentinelone-login": {
+    "fields": {
+      "machine_ip": [
+        "srcMachineIP"
+      ],
+      "login_type": [
+        "loginType"
+      ],
+      "account_domain": [
+        "loginAccountDomain"
+      ]
+    }
+  },
+  "x-sentinelone-file": {
+    "fields": {
+      "file_type": [
+        "tgtFileType"
+      ],
+      "extention": [
+        "tgtFileExtension"
+      ],
+      "description": [
+        "tgtFileDescription"
+      ],
+      "location": [
+        "tgtFileLocation"
+      ]
+    }
+  },
+  "x-sentinelone-registry": {
+    "fields": {
+      "path": [
+        "registryPath"
+      ],
+      "full_size": [
+        "registryValueFullSize"
+      ]
+    }
+  },
+  "x-sentinelone-indicator": {
+    "fields": {
+      "indicator_name": [
+        "indicatorName"
+      ],
+      "indicator_category": [
+        "indicatorCategory"
+      ],
+      "indicator_description": [
+        "indicatorDescription"
+      ],
+      "indicator_metadata": [
+        "indicatorMetadata"
+      ]
+    }
+  },
+  "x-oca-asset": {
+    "fields": {
+      "hostname": ["endpointName"],
+      "ip_refs[*].value": ["srcIp", "dstIp"]
+    }
+  },
+  "x-oca-endpoint": {
```

Comment:
Endpoint wasn't defined as a custom STIX extension definition. This could likely go under `x-sentinelone-endpoint`.

Reply:
Now extensions field are included inside "x-oca-asset" object. Similarly we have changed for process, file, network-traffic and other fields.